recipes/roounfold/recipe.yaml

```
@@ -0,0 +1,72 @@
+context:
+  name: RooUnfold
+  version: "3.0.5"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://gitlab.cern.ch/RooUnfold/RooUnfold/-/archive/${{ version }}/${{ name }}-${{ version }}.tar.gz
+  sha256: e57d846efcaff831dfeaad528fa9ebc60fe3e09165decbbc261592c23851d40a
+  patches:
+    - patches/cmake-minimum-3.5.patch
+
+build:
+  number: 0
+  script:
+    - cmake ${CMAKE_ARGS}  -S . -B build
+    - cmake --build build --clean-first --parallel="${CPU_COUNT}"
+    - cmake --install build
+
+requirements:
+  build:
+    - ${{ compiler("cxx") }}
+    #- ${{ stdlib("c") }}
```

```suggestion
    - ${{ stdlib("c") }}
```

All recipes that use a compiler are required to use the `stdlib("c")` metapcakge.

c.f. https://conda-forge.org/news/2024/03/24/stdlib-migration/

---

accidental commit due to me running locally and dealing with https://github.com/prefix-dev/rattler-build/issues/1794